src/applications/appeals/10182/validations/index.js

```diff
@@ -1,16 +1,16 @@
 import { areaOfDisagreementWorkAround } from '../utils/ui';
 import { missingAreaOfDisagreementErrorMessage } from '../content/areaOfDisagreement';
 
-export const contactInfoValidation = (errors, _fieldData, formData) => {
-  const { veteran = {}, homeless } = formData;
+export const contactInfoValidation = (errors = {}, _fieldData, formData) => {
+  const { veteran = {}, homeless } = formData || {};
   if (!veteran.email) {
-    errors.addError('Please add an email address to your profile');
+    errors.addError?.('Please add an email address to your profile');
   }
   if (!veteran.phone?.phoneNumber) {
-    errors.addError('Please add a phone number to your profile');
+    errors.addError?.('Please add a phone number to your profile');
```

Comment:
So I think what's happening is that the react json schema form library is initializing, or something, so that the `errors` object isn't defined. These validation functions are called numerous times, so even if it doesn't add an error one time, it usually ends up capturing the error. Worse case scenario the error isn't shown and the Veteran can get past the contact info page, but prior to submission, a separate validation library is run and will prevent the network call. And the accordion will highlight the issue.